packages/runtime/test/src/interpreter/inner-transaction.ts

```diff
@@ -1396,5 +1408,45 @@ describe("Inner Transactions", function () {
 				);
 			});
 		});
+
+		describe("Foreign application account should be accessed in teal v7 ", () => {
+			this.beforeEach(() => {
+				setUpInterpreter(7, 1e9);
+			});
+
+			it("Should succeed: create inner group transactions", () => {
```

Comment:
Should set the receiver of the application to foreign app account?

Reply:
yes we should this is the test case I cover from go-algorand too!

Reply:
That was a suggestion for renaming the test

Reply:
So I should rename the test right?

Reply:
Lets make it more descriptive. Try explaining what should actually happen